packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -151,6 +150,9 @@ export const ChatList = ({
     chatListClient.onStateChange(state => {
       if (state.status === 'chat message received') {
         if (onMessageReceived && state.chatMessage) {
+          if (state.chatMessage.chatId === internalSelectedChatId) {
```

Comment:
one change I am doing is to move state changes down to `StatefulGraphChatListClient`. Now, we mostly have eventing on this layer if you want to refer to my PR